gpytorch/lazy/kronecker_sum_lazy_tensor.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+from .. import settings
+from .kronecker_product_lazy_tensor import KroneckerProductLazyTensor
+from .sum_lazy_tensor import SumLazyTensor
+
+
+class KroneckerSumLazyTensor(SumLazyTensor):
+    r"""
+    Returns the sum of two Kronecker product lazy tensors. Solves and log-determinants
+    are computed using the eigen-decomposition of the right lazy tensor; that is,
+    (A \kron B + C \kron D) = (C^{1/2} \kron D^{1/2})
+        (C^{-1/2}AC^{-1/2} \kron D^{-1/2} B D^{-1/2} + I_|C| \kron I_|D|)(C^{1/2} \kron D^{1/2})^{T}
+    where .^{1/2} = Q_.\Lambda_.^{1/2} (e.g. an eigen-decomposition.)
+
+    This formulation admits efficient solves and log determinants.
+
+    The original reference is [https://papers.nips.cc/paper/2013/file/59c33016884a62116be975a9bb8257e3-Paper.pdf].
+
+    Args:
+        :`lazy_tensors`: List of lazy tensors
+    """
```

Comment:
It looks like this LT only supports summing  2 LTs, not a variable number? Should we add an explicit constructor for it?